solutions/advanced/Ligatures.mdx

```diff
@@ -0,0 +1,171 @@
+---
+id: kattis-ligatures
+source: Kattis
+title: Ligatures
+author: Alexis Tao
+---
+
+### Explanation
+
+The problem involves counting the occurrences of specific pairs of characters (ligatures) in a given text corpus based on multiple sets of suggested ligatures. The primary challenge is efficiently handling large input sizes and ensuring that the counting process respects the rule that ligatures can't overlap.
```

Comment:
is there a focus problem link

Reply:
not a focus problem, but it's linked here: https://usaco-guide-18mueqdak-cpinitiative.vercel.app/adv/vectorization#problems

Reply:
i mean if it's the problem the module goes over...shouldn't it be put as one instead?

Reply:
it's not supposed to be an in-module problem

Reply:
wait how does that even work

Reply:
how does what work?

Reply:
when would a problem not be in-module? i don't think any other module has this kind of thing...

Reply:
I don't think there's any competitive programming problem like this one 🤷. It feels strange to add it in-module with no examples after it.